### PR TITLE
BIG-17565 Add function to remove a header from Connection object.

### PR DESCRIPTION
**Pull request by bc-ravijayaramappa**
@lord2800

### CODE REVIEW

File: src/Bigcommerce/Api/Connection.php [hunk 1]
```diff
@@ -165,6 +165,17 @@ public function addHeader($header, $value)
     }
 
     /**
+     * Remove a header from the request.
+     * @param string $header
+     */
+    public function removeHeader($header)
+    {
+        if (!empty($header) && isset($this->headers[$header])) {
```
Comment (lord2800) [inline src/Bigcommerce/Api/Connection.php]: You only need the `isset` check. Technically you don't even need that.

**Comment by lord2800** [comment]
:+1:

**Comment by lord2800** [comment]
Seems like github and travis don't agree right now, so I'll wait to merge.

**Comment by lord2800** [comment]
Just awaiting Travis now.

**Comment by aleachjr** [comment]
Please do not merge this without adding a test unless you are unable to write one.

**Comment by lord2800** [comment]
This is adding a method to the `Connection` object, which isn't testable at this time.

**Comment by bc-ravijayaramappa** [comment]
@aleachjr Merge please?

**Pull request merged**